src/main/java/uk/gov/hmcts/darts/task/service/impl/AdminAutomatedTasksServiceImpl.java

```diff
@@ -68,6 +74,8 @@ public void runAutomatedTask(Integer taskId) {
         }
 
         automatedTaskRunner.run(automatedTask.get());
+
+        auditApi.record(RUN_JOB_MANUALLY, authorisationApi.getCurrentUser(), null);;
```

Comment:
Theres an overloaded api call `record(activity)`that will default to the current user and use null for the case

Reply:
np. will make the change